contracts/vaults/NonvotingVault.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.3;
+
+import "../libraries/Authorizable.sol";
+import "../interfaces/ILockingVault.sol";
+import "../interfaces/IERC20.sol";
+
+contract NonvotingVault is Authorizable {
+    // The locking gov vault
+    ILockingVault public lockingVault;
+    IERC20 public immutable token;
+
+    /// @notice Constructs this contract
+    /// @param _owner The address authorized to withdraw
+    /// @param _lockingVault The governance vault which this withdraws from
+    constructor(
+        address _owner,
+        IERC20 _token,
+        ILockingVault _lockingVault
+    ) Authorizable() {
+        setOwner(_owner);
+        lockingVault = _lockingVault;
+        token = _token;
```

Comment:
To ensure that the token matches the locking vault either (1) we should add a require that it matches `lockingVault.token()` or (2) set `token = lockingVault.token()` directly.

Reply:
went with option 2!